contrib/ansible/roles/skydive_dev/tasks/lxd.yml

```diff
@@ -6,6 +6,7 @@
 - package:
     name: yum-plugin-copr
     state: present
+  when: ansible_distribution != "Fedora"
```

Comment:
Why not on Fedora? 
What are you trying to solve?

Reply:
The creation of vagrant boxes is currently broken because `yum-plugin-copr` conflicts with `dnf` with has been the default on Fedora for a while now.